config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -46,6 +46,11 @@ data:
       - "InvalidSubnet: Not enough IP space available in"
       installFailingReason: AWSSubnetInsufficientIPSpace
       installFailingMessage: Insufficient IP space available in subnet
+    - name: AWSSubnetTagLimitExceeded
+      searchRegexStrings:
+      - "could not add tags to subnets: TagLimitExceeded"
+      installFailingReason: AWSSubnetTagLimitExceeded
+      installFailingMessage: AWS Subnet exceeds the maximum number of tags allowed (50)
```

Comment:
How future-proof is this message? Will it always be 50? Is the limit tunable (like a quota)?

Reply:
The tag limit is not tunable